toolchain/check/testdata/impl/no_prelude/name_poisoning.carbon

```diff
@@ -17,10 +17,10 @@ interface I;
 // `N.F` uses `N.I` and not `package.I`.
 namespace N;
 interface N.I {}
-fn N.F(x: I*);
+fn N.F(x:! I) {}
 
-fn TestCall(x: N.I*) {
-  // `N.F` accepts an `N.I*` not a `package.I*`.
+fn TestCall(x:! N.I) {
+  // `N.F` accepts an `N.I` not a `package.I`.
```

Comment:
Could you also change the other tests where you use pointers to interfaces to use generics instead?

Reply:
Done.